api/v1beta1/gitrepository_types.go

```diff
@@ -44,7 +44,7 @@ type GitRepositorySpec struct {
 	// The secret name containing the Git credentials.
 	// For HTTPS repositories the secret must contain username and password
 	// fields.
-	// For SSH repositories the secret must contain identity, identity.pub and
```

Comment:
`v1beta1` has gone stale and does not require further modifications.

Reply:
Will remove the change. Does this also apply to v1alpha1?

Reply:
Yes, anything that isn't current.